repos.yaml

```diff
@@ -0,0 +1,14 @@
+# This file contains the contact information for repositories exposed through
+# the Helm Hub. Each repository has a url, the base path to its location, and
+# one or more maintainers.
+#
+# The structure of the repositories object is:
+# repositories:
+#  - url: https://example.com
+#    maintainers:
+#      - name: foo
+#        email: foo@example.com
+#
+# Note, for the email address it is acceptable to have a mailing list for a team
+# because projects often have multiple people who may rotate.
+repositories:
```

Comment:
should we seed this with an example for the stable and incubator repos? We can use one of the Helm mailing list emails?

Reply:
Do we have a charts email address that is available for the public to email? I don't know the settings on the one we have.

Reply:
I don't think the current one is public to emails, is there another Helm one we can use? I'm just thinking it would be good to have here since we state that each repo in repo-values.yaml should have a corresponding entry in repos.yaml, but that isn't currently the case. We can figure that out in a separate PR though!